chef/cookbooks/ntp/recipes/default.rb

```diff
@@ -91,7 +91,7 @@
   service "ntp" do
     service_name "ntpd" if node[:platform_family] == "rhel"
     service_name "ntpd" if node[:platform] == "opensuse" || (node[:platform] == "suse" && node[:platform_version].to_f >= 12.0)
-    supports restart: true, status: true, reload: true
+    supports restart: true, status: true, reload: false
```

Comment:
FWIW, I'd simply remove that `supports` line. Chef will do the right thing without it.